src/server.cpp

```diff
@@ -1191,12 +1204,22 @@ void CServer::MixEncodeTransmitData ( const int iChanCnt,
     else
     {
         // Stereo target channel -----------------------------------------------
+
+        const int maxPanDelay = MAX_DELAY_PANNING_SAMPLES; // 32  -->  64
+
+        int iLpan, iRpan, iPan;
+
         for ( j = 0; j < iNumClients; j++ )
         {
             // get a reference to the audio data and gain/pan of the current client
             const CVector<int16_t>& vecsData = vecvecsData[j];
+            const CVector<int16_t>& vecsData2 = vecvecsData2[j];
+
             const float             fGain    = vecvecfGains[iChanCnt][j];
-            const float             fPan     = vecvecfPannings[iChanCnt][j];
+            const float             fPan     = bDelayPan ? 0.5f : vecvecfPannings[iChanCnt][j];
+            const int iPanDel = lround( (float)(2 * maxPanDelay - 2) * (vecvecfPannings[iChanCnt][j] - 0.5f) );
```

Comment:
I'm not sure about the style here.

```suggestion
            const int iPanDel = lround( (float)( 2 * maxPanDelay - 2 ) * ( vecvecfPannings[iChanCnt][j] - 0.5f ) );
```

Does this line calculate the delay between left and right? Sorry if this is obvious ;-).

Reply:
I am not sure. There are more places where you could save on temporary variables. So far I haven't really understood what's been done here.

Reply:
Hmm. Before merging this, we (or at least one/two of the main developers) have to be exactly sure what each line does (and comment it). If anything goes wrong it should be possible to debug it.

Reply:
Having investigated the code more `iPanDel` is the position and the delay happens later by adding `iServerFrameSizeSamples` to one of the two channels. Correct me if I’m wrong. 

We should also check how `maxPanDelay` is set.